<?xml version="1.0" encoding="UTF-8"?>
<testng-results ignored="0" total="1" passed="0" failed="1" skipped="0">
  <reporter-output>
    <line>
      <![CDATA[User logged in successfully]]>
    </line>
    <line>
      <![CDATA[Workflows Tab is clicked]]>
    </line>
    <line>
      <![CDATA[Import button clicked]]>
    </line>
    <line>
      <![CDATA[assisted Workflow is selected]]>
    </line>
    <line>
      <![CDATA[assisted Workflow is unselected]]>
    </line>
    <line>
      <![CDATA[Create Button is clicked]]>
    </line>
    <line>
      <![CDATA[Priority is set]]>
    </line>
    <line>
      <![CDATA[Expected Completion Time in Seconds is set]]>
    </line>
    <line>
      <![CDATA[Maximum Completion Time in Seconds is set]]>
    </line>
    <line>
      <![CDATA[Cleanup Requests older than Hours fields is set]]>
    </line>
    <line>
      <![CDATA[Manual Execution Time is set]]>
    </line>
    <line>
      <![CDATA[Manual Execution time unit is set]]>
    </line>
    <line>
      <![CDATA[Save button is clicked]]>
    </line>
    <line>
      <![CDATA[Workflow updated]]>
    </line>
    <line>
      <![CDATA[Workflow is enabled successfully]]>
    </line>
    <line>
      <![CDATA[User signed out successfully]]>
    </line>
    <line>
      <![CDATA[User logged in successfully]]>
    </line>
    <line>
      <![CDATA[Catalogue Tab is clicked]]>
    </line>
    <line>
      <![CDATA[Workflow whose request needs to be send is selected]]>
    </line>
    <line>
      <![CDATA[Submit request button is clicked]]>
    </line>
    <line>
      <![CDATA[Actual Popup Box:- Failure!]]>
    </line>
    <line>
      <![CDATA[Expected Popup Box:- Success!]]>
    </line>
  </reporter-output>
  <suite started-at="2023-02-02T07:22:01Z" name="Default suite" finished-at="2023-02-02T07:23:21Z" duration-ms="79433">
    <groups>
    </groups>
    <test started-at="2023-02-02T07:22:01Z" name="Default test" finished-at="2023-02-02T07:23:21Z" duration-ms="79433">
      <class name="com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA">
        <test-method is-config="true" signature="setExtent()[pri:0, instance:com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA@7b64240d]" started-at="2023-02-02T07:22:01Z" name="setExtent" finished-at="2023-02-02T07:22:02Z" duration-ms="189" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- setExtent -->
        <test-method is-config="true" signature="initialization()[pri:0, instance:com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA@7b64240d]" started-at="2023-02-02T07:22:02Z" name="initialization" finished-at="2023-02-02T07:22:07Z" duration-ms="5259" status="PASS">
          <reporter-output>
          </reporter-output>
        </test-method> <!-- initialization -->
        <test-method signature="validateSubmitReqCustomCategoryWATest(java.lang.reflect.Method)[pri:106, instance:com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA@7b64240d]" started-at="2023-02-02T07:22:07Z" name="validateSubmitReqCustomCategoryWATest" finished-at="2023-02-02T07:23:12Z" duration-ms="64855" status="FAIL">
          <params>
            <param index="0">
              <value>
                <![CDATA[public void com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA.validateSubmitReqCustomCategoryWATest(java.lang.reflect.Method) throws java.lang.Exception]]>
              </value>
            </param>
          </params>
          <exception class="java.lang.AssertionError">
            <message>
              <![CDATA[Popup Box Message is Failure! expected [Success!] but found [Failure!]]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.AssertionError: Popup Box Message is Failure! expected [Success!] but found [Failure!]
at org.testng.Assert.fail(Assert.java:96)
at org.testng.Assert.failNotEquals(Assert.java:776)
at org.testng.Assert.assertEqualsImpl(Assert.java:137)
at org.testng.Assert.assertEquals(Assert.java:118)
at org.testng.Assert.assertEquals(Assert.java:453)
at com.ae.qa.pages.WorkflowAdmin.CataloguePageWA.validateSubmitReqCustomCategory(CataloguePageWA.java:235)
at com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA.validateSubmitReqCustomCategoryWATest(CataloguePageTestWA.java:34)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.Invoker.invokeMethod(Invoker.java:583)
at org.testng.internal.Invoker.invokeTestMethod(Invoker.java:719)
at org.testng.internal.Invoker.invokeTestMethods(Invoker.java:989)
at org.testng.internal.TestMethodWorker.invokeTestMethods(TestMethodWorker.java:125)
at org.testng.internal.TestMethodWorker.run(TestMethodWorker.java:109)
at org.testng.TestRunner.privateRun(TestRunner.java:648)
at org.testng.TestRunner.run(TestRunner.java:505)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.AssertionError -->
          <reporter-output>
            <line>
              <![CDATA[User logged in successfully]]>
            </line>
            <line>
              <![CDATA[Workflows Tab is clicked]]>
            </line>
            <line>
              <![CDATA[Import button clicked]]>
            </line>
            <line>
              <![CDATA[assisted Workflow is selected]]>
            </line>
            <line>
              <![CDATA[assisted Workflow is unselected]]>
            </line>
            <line>
              <![CDATA[Create Button is clicked]]>
            </line>
            <line>
              <![CDATA[Priority is set]]>
            </line>
            <line>
              <![CDATA[Expected Completion Time in Seconds is set]]>
            </line>
            <line>
              <![CDATA[Maximum Completion Time in Seconds is set]]>
            </line>
            <line>
              <![CDATA[Cleanup Requests older than Hours fields is set]]>
            </line>
            <line>
              <![CDATA[Manual Execution Time is set]]>
            </line>
            <line>
              <![CDATA[Manual Execution time unit is set]]>
            </line>
            <line>
              <![CDATA[Save button is clicked]]>
            </line>
            <line>
              <![CDATA[Workflow updated]]>
            </line>
            <line>
              <![CDATA[Workflow is enabled successfully]]>
            </line>
            <line>
              <![CDATA[User signed out successfully]]>
            </line>
            <line>
              <![CDATA[User logged in successfully]]>
            </line>
            <line>
              <![CDATA[Catalogue Tab is clicked]]>
            </line>
            <line>
              <![CDATA[Workflow whose request needs to be send is selected]]>
            </line>
            <line>
              <![CDATA[Submit request button is clicked]]>
            </line>
            <line>
              <![CDATA[Actual Popup Box:- Failure!]]>
            </line>
            <line>
              <![CDATA[Expected Popup Box:- Success!]]>
            </line>
          </reporter-output>
        </test-method> <!-- validateSubmitReqCustomCategoryWATest -->
        <test-method is-config="true" signature="getResult(org.testng.ITestResult)[pri:0, instance:com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA@7b64240d]" started-at="2023-02-02T07:23:12Z" name="getResult" finished-at="2023-02-02T07:23:12Z" duration-ms="328" status="PASS">
          <params>
            <param index="0">
              <value>
                <![CDATA[[TestResult name=validateSubmitReqCustomCategoryWATest status=FAILURE method=CataloguePageTestWA.validateSubmitReqCustomCategoryWATest(java.lang.reflect.Method)[pri:106, instance:com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA@7b64240d] output=User logged in successfully]]]>
              </value>
            </param>
          </params>
          <reporter-output>
          </reporter-output>
        </test-method> <!-- getResult -->
        <test-method is-config="true" signature="endReport()[pri:0, instance:com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA@7b64240d]" started-at="2023-02-02T07:23:12Z" name="endReport" finished-at="2023-02-02T07:23:21Z" duration-ms="8764" status="FAIL">
          <exception class="java.lang.RuntimeException">
            <message>
              <![CDATA[javax.mail.AuthenticationFailedException: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect. [PN3PR01CA0046.INDPRD01.PROD.OUTLOOK.COM 2023-02-02T07:23:21.099Z 08DB04789F5E93DD]
]]>
            </message>
            <full-stacktrace>
              <![CDATA[java.lang.RuntimeException: javax.mail.AuthenticationFailedException: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect. [PN3PR01CA0046.INDPRD01.PROD.OUTLOOK.COM 2023-02-02T07:23:21.099Z 08DB04789F5E93DD]

at com.ae.qa.util.SendMail.sendExtentReport(SendMail.java:79)
at com.ae.qa.base.TestBase.endReport(TestBase.java:218)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:78)
at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
at java.base/java.lang.reflect.Method.invoke(Method.java:567)
at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:124)
at org.testng.internal.MethodInvocationHelper.invokeMethodConsideringTimeout(MethodInvocationHelper.java:59)
at org.testng.internal.Invoker.invokeConfigurationMethod(Invoker.java:458)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:222)
at org.testng.internal.Invoker.invokeConfigurations(Invoker.java:142)
at org.testng.TestRunner.afterRun(TestRunner.java:725)
at org.testng.TestRunner.run(TestRunner.java:509)
at org.testng.SuiteRunner.runTest(SuiteRunner.java:455)
at org.testng.SuiteRunner.runSequentially(SuiteRunner.java:450)
at org.testng.SuiteRunner.privateRun(SuiteRunner.java:415)
at org.testng.SuiteRunner.run(SuiteRunner.java:364)
at org.testng.SuiteRunnerWorker.runSuite(SuiteRunnerWorker.java:52)
at org.testng.SuiteRunnerWorker.run(SuiteRunnerWorker.java:84)
at org.testng.TestNG.runSuitesSequentially(TestNG.java:1208)
at org.testng.TestNG.runSuitesLocally(TestNG.java:1137)
at org.testng.TestNG.runSuites(TestNG.java:1049)
at org.testng.TestNG.run(TestNG.java:1017)
at org.testng.remote.AbstractRemoteTestNG.run(AbstractRemoteTestNG.java:115)
at org.testng.remote.RemoteTestNG.initAndRun(RemoteTestNG.java:251)
at org.testng.remote.RemoteTestNG.main(RemoteTestNG.java:77)
Caused by: javax.mail.AuthenticationFailedException: 535 5.7.139 Authentication unsuccessful, the user credentials were incorrect. [PN3PR01CA0046.INDPRD01.PROD.OUTLOOK.COM 2023-02-02T07:23:21.099Z 08DB04789F5E93DD]

at com.sun.mail.smtp.SMTPTransport$Authenticator.authenticate(SMTPTransport.java:965)
at com.sun.mail.smtp.SMTPTransport.authenticate(SMTPTransport.java:876)
at com.sun.mail.smtp.SMTPTransport.protocolConnect(SMTPTransport.java:780)
at javax.mail.Service.connect(Service.java:388)
at javax.mail.Service.connect(Service.java:246)
at javax.mail.Service.connect(Service.java:195)
at javax.mail.Transport.send0(Transport.java:254)
at javax.mail.Transport.send(Transport.java:124)
at com.ae.qa.util.SendMail.sendExtentReport(SendMail.java:76)
... 25 more
]]>
            </full-stacktrace>
          </exception> <!-- java.lang.RuntimeException -->
          <reporter-output>
          </reporter-output>
        </test-method> <!-- endReport -->
      </class> <!-- com.ae.qa.testcasesWorkflowAdmin.CataloguePageTestWA -->
    </test> <!-- Default test -->
  </suite> <!-- Default suite -->
</testng-results>
